### PR TITLE
Refactor '_packPPTPayload' to reduce cognitive complexity

### PR DESCRIPTION
**Pull request by lucasrmendonca**
### Description, Motivation and Context
Refactor '_packPPTPayload' to reduce cognitive complexity

### What kind of change does this PR introduce?
- [x] Refactoring (no new functionality, only code improvements)

### Checklist:

- [x] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [x] Overall test coverage is not decreased.
- [x] All new and existing tests passed.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/wampy.js [hunk 1]
```diff
@@ -559,81 +559,73 @@ class Wampy {
...
-            if (options.ppt_serializer && options.ppt_serializer !== 'native') {
-                const pptSerializer = this._options.payloadSerializers[options.ppt_serializer];
+        if (isPayloadAnObject && !argsList && !argsDict) {
+            kwargs = payload;
+        } else if (isPayloadAnObject) {
+            args = argsList;
+            kwargs = argsDict;
+        } else {
+            args = Array.isArray(payload) ? payload : [payload];
+        }
```
Comment (KSDaemon) [inline src/wampy.js:582]: Mb change it to this, so we can keep a simple one-level if-else block without inner checks which we can not avoid.

```suggestion
        } else if (Array.isArray(payload)) {
            args = payload;
        } else {  // assume it's a single value
            args = [payload];
        }
```

**Comment by lucasrmendonca** [comment]
Uh-oh I might need some help figuring out what happened to the tests after I commited your suggestion :(

**Comment by KSDaemon** [comment]
Nothing special — just was a bad moon position for node v12 :)
Hint: In such cases you can always try to rerun failed jobs. If they fail again — then it is a reason to look into our code. But sometimes they just fail with no particular reason

**Review by KSDaemon — APPROVED** [review]
👍🏻

**Comment by lucasrmendonca** [comment]
> Nothing special — just was a bad moon position for node v12 :) Hint: In such cases you can always try to rerun failed jobs. If they fail again — then it is a reason to look into our code. But sometimes they just fail with no particular reason

Thanks!

**Comment by KSDaemon** [comment]
👍🏻

**Pull request merged**